react/javascript/tile-sdk/src/components/Inspector/components/EventTester/EventTester.js

```diff
@@ -97,11 +99,31 @@ export const EventTester = () => {
   )
 
   const openDrillMenuClick = useCallback(
-    (event) => {
-      // TODO links data needs to be populated
-      tileSDK.openDrillMenu({ links: [] }, event)
+    (_event) => {
+      let event = { pageX: 0, pageY: 0 }
+      let links = []
+      if (visualizationData?.queryResponse?.data) {
+        const data = visualizationData?.queryResponse?.data
```

Comment:
very minor nit but couldn't you assign this outside the `if` and then check it for the conditional logic?

Reply:
Yep, I like that! Done